omc3/old_measurement_converter.py

```diff
@@ -0,0 +1,206 @@
+"""
```

Comment:
Could use a description. And should probably be in `scripts`

Reply:
Should not be in here at all...it is in separate branch for separate issue: https://github.com/pylhc/omc3/commit/3f05be47a88ebbedeb4f0845986696af4f376e5e

Reply:
@lmalina do you confirm we can delete the file from this branch?